scripts/build.sh

```diff
@@ -39,8 +39,6 @@ if version_lt "$(go_version)" "$go_version_minimum"; then
     exit 1
 fi
 
-scripts/protobuf_codegen.sh
```

Comment:
This was causing some issues locally. We shouldn't need to install buf to run anything, since the files it generates are checked in.

Reply:
If we remove this then we should remove the README instruction to install buf, under `Building`.

Also though, I think the README should also mention/explain that we have a codgen stage, and point to the scripts we use for that.